lib/rails_api_format/lib/rails_api_format/error.rb

```diff
@@ -0,0 +1,25 @@
+module RailsApiFormat
+  class Error
+    include ActiveModel::Model
+    include ActiveModel::Serialization
+
+    ATTRIBBUTES = %i(status error validations)
+    attr_accessor(*ATTRIBBUTES)
```

Comment:
it seems like we use `ATTRIBUTES` only for defining attr accessors or have I missed something?